cf/test/test_collapse.py

```diff
@@ -187,6 +187,20 @@ def test_Field_collapse_CLIMATOLOGICAL_TIME(self):
             print(g.constructs)
         self.assertEqual(list(g.shape), expected_shape)
 
+        g = f.collapse(
+            "T: max within years time: minimum over years",
+            within_years=cf.seasons(),
+            over_years=cf.Y(1),
+        )
+        expected_shape = list(f.shape)
+        expected_shape[0] = 12
+
+        if verbose:
```

Comment:
We should probably use a more consolidated way to adjust verbosity in the tests in future (via logging or the messages), but for now this does the job!

Reply:
Agreed.